cli/src/customize/mod.rs

```diff
@@ -258,10 +266,18 @@ impl CustomizableFields {
 }
 
 pub trait Customize {
+    /// Customize the program with the given options.
     fn customize(&self, program: Program<CBNCache>) -> CliResult<Program<CBNCache>>;
+    /// Return the value of the `field` argument, if specified and if supported by the current
+    /// customize variant.
+    fn field(&self) -> Option<&String> {
```

Comment:
Is there a reason to use `&String` here instead of `&str`?

Reply:
Not really. In fact we don't even use the content, what I use it for is to check if `field` is set in `nickel query` and issue a warning if not. Maybe the interface should then be minimal, that is `is_field_set(&self) -> bool`, which eschews the question entirely